src/main/java/cloud/eppo/ufc/dto/EppoValue.java

```diff
@@ -107,4 +108,25 @@ public String toString() {
             "Cannot stringify Eppo Value type " + this.type.name());
     }
   }
+
+  @Override
+  public boolean equals(Object otherObject) {
```

Comment:
Overriding `equals()` and `hashCode()` allows for the comparison of different instances of Eppo values based on the value they contain. This is particularly useful for `assertEquals()` in automated tests.